gnovm/pkg/gnolang/realm.go

```diff
@@ -1121,6 +1121,10 @@ func copyValueWithRefs(parent Object, val Value) Value {
 		}
 	case *FuncValue:
 		source := toRefNode(cv.Source)
+		if strings.HasSuffix(source.Location.File, "_test.gno") {
```

Comment:
```suggestion
		func (v Val) InTestFile() bool {return  strings.HasSuffix(source.Location.File, "_test.gno")}
		if val.InTestFile()) {
```

Reply:
We don't have to change it now.  I created an issue so that we can track it for later discussion. 

Dose this mean if someone  deployed a realm package  with a source file named xyz_test.gno it will be ignored by the VM?   If that is the case, it means we should include this file naming convention to the VM specs.  

Also If we do not allow _test.gno files in the realm package, we should guarded it before we parse the file and load it to VM, instead of at the persistent stage.  

If this purely serves  the purpose for off chain realm testing,  would sticking with  realm file test be a better option?